### ISSUE TITLE
Leading edge of reflected light waves appear to speed up when reflecting object removed 

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#269]
I found this when doing some sanity testing for #268.  On the waves screen, if the cloud is switched on and off quickly, or if the scene with the glacier is switched in and out quickly, the leading edge of the wave starts moving faster.  The trailing edge does not, so the the wave fragment appears to get longer.

Below is a screenshot where the part that is moving too fast is marked.  A video would be better, but I'm not very good at those, and it doesn't seem necessary.

![image](https://user-images.githubusercontent.com/3082831/219513832-ff5b9b14-21ae-4adc-a81e-5e2574c96cbc.png)

**Comment by jbphet** [comment]
I've investigated this and have submitted a fix.  This problem appears to have been introduced when the intensity changes were added for waves in issue https://github.com/phetsims/greenhouse-effect/issues/84.  That means the bug was there since Sep 2022, so it's surprising that it slipped through undetected, but it is fairly subtle.

This issue exists with the published prototype, version 1.1.1.  I'd like to test a bit more before propagating it to the release branch, and if all is well I will put it into the soon-to-be-published version.
